content/learning-paths/servers-and-cloud-computing/cca-veraison/_next-steps.md

```diff
@@ -0,0 +1,28 @@
+---
+next_step_guidance: Now that you have gained some hands-on experience with the data formats and workflows associated with attestation for confidential computing, you may wish to explore some additional resources and specifications, which go into greater detail on some of the individual aspects.
+
+recommended_path: /learning-paths/PLACEHOLDER_CATEGORY/PLACEHOLDER_LEARNING_PATH/
+
+
+further_reading:
```

Comment:
Looks like we haven't put the further reading links in. Some recommendations were in the original draft content that I provided.

Reply:
done